Backpack/Calendar/Classes/BPKCalendarPriceLabelConfiguration.m

```diff
@@ -29,7 +29,7 @@ - (instancetype)init {
     // 2 is the spacing between the title and the price label
     // 29 is the height of the price label when wrapped onto 2 lines
     // 8 is the spacing between the price label and the cell bottom
-    return [super initWithCellClass:BPKCalendarPriceLabelCell.class cellDataClass:BPKCalendarPriceLabelCellData.class rowHeight:70];
+    return [super initWithCellClass:BPKCalendarPriceLabelCell.class cellDataClass:BPKCalendarPriceLabelCellData.class rowHeight:74];
```

Comment:
As the base cell size is larger, the price cells also need to be larger

Reply:
Do we also need to update the comment for this so that the number now correctly displays 74 and how its calculated?

Reply:
Oohh good catch. Also `29` and `31` should have been `28` and `32` before 🤦‍♀️

Reply:
Can we make this a constant that is based on the two ohter constants? i.e. instead of a comment + hard coded numbers we can have the actual math with constants?